core/orch_test.go

```diff
@@ -1207,13 +1207,13 @@ func TestIsActive(t *testing.T) {
 	}
 	orch := NewOrchestrator(n, rm)
 
-	ok, err := orch.isActive(addr)
+	ok, err := orch.isPaymentEligible(addr)
```

Comment:
WDYT about re-naming `TestIsActive()` to `TestIsPaymentEligible`?

Reply:
Updated in https://github.com/livepeer/go-livepeer/pull/2146/commits/07a2560b3df9680245cbe83ecfc905f727449d19